src/scss/grommet-core/_objects.video.scss

```diff
@@ -7,6 +7,10 @@
   @include media-query(palm) {
     max-width: 100%;
     width: 100vw;
+
+    &__title {
+      visibility: hidden;
```

Comment:
The visibility hidden there prevents the control button from being pushed to the bottom of the video summary overlay on mobile.